### PR TITLE
1.19.2

### PR DESCRIPTION
**Pull request by electriquo**
added retry-after header (429 status code) to limit_req

### CODE REVIEW

**Comment by tsuyoshizawa** [comment]
@y0y0z Thank you for a your PR.

This PR is getting conflicts because of we merged #8 first.
Could you fix the conflict?

**Comment by electriquo** [comment]
@tsuyoshizawa:
conflict fixed, but please do let me know if the conflict still exists. 
would appreciate your comment on https://github.com/nulab/nginx-too-many-requests-retryafter-patch/issues/10

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ The following versions of Nginx should work with this module:
 
 Module version | Nginx version
 --- | ---
+1.3.0 | 1.19.2 or higher
```
Comment (tsuyoshizawa) [inline README.md]: Does it have to be a specific version of `1.19.2` instead of `1.19.x`?

Comment (electriquo) [reply README.md]: testing it for 1.19.x (1.19.0, 1.19.1) as we speak, will post a fix in a few.

Comment (electriquo) [reply README.md]: it works for 1.19.x.
i've pushed a changed to the README.

**Pull request merged**